app/views/chargeback/_cb_rate_edit.html.haml

```diff
@@ -26,7 +26,12 @@
       .col-md-8
         = select_tag("currency",
           options_for_select(currency, @edit[:new][:currency]),
-          "data-miq_observe" => {:url => url}.to_json)
+          "data-miq_observe" => {:url => url}.to_json,
```

Comment:
Can't use `data-miq_observe` and `miqSelectPickerEvent` at the same time - need to remove this line ;)